### PR TITLE
About page: Contributors

### PR DESCRIPTION
**Pull request by onnovisser**
image
image

#465

### CODE REVIEW

File: src/components/ContributorsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+          rel="noopener noreferrer"
+          target="_blank"
+          aria-label="Social media"
+          position="absolute"
+          top={[0, 'initial']}
+          left={[0, 'initial']}
+          bottom={['initial', 0]}
+          right={['initial', 0]}
+        >
+          <Box as="img" display="block" src={link} maxWidth={[24, 30]} />
```
Comment (Hornebom) [inline src/components/ContributorsSection.tsx:61]: You can use the `<Image data={image}  alt={alt} />` component

File: src/components/ContributorsSection.tsx [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    [`@media (min-width: ${props.theme.breakpoints['S']})`]: {
+      transform: 'none !important',
+    },
+    [`@media (max-width: ${props.theme.breakpoints['S']})`]: {
+      [`& > :not(:nth-child(${props.$pages}n + ${props.$page + 1}))`]: {
+        opacity: 0,
+      },
+    },
+  })
+)
```
Comment (Hornebom) [inline src/components/ContributorsSection.tsx:96]: How do you feel about extracting those in a `styles.tsx` file?

File: src/components/ContributorsSection.tsx [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+                <Button
+                  variant="tertiary"
+                  icon={IconArrowLeft}
+                  onClick={() => setPage((prev) => (prev - 1 + numPages) % numPages)}
+                />
+                <Button
+                  variant="tertiary"
+                  icon={IconArrowRight}
+                  onClick={() => setPage((prev) => (prev + 1) % numPages)}
+                />
```
Comment (Hornebom) [inline src/components/ContributorsSection.tsx:144]: add descriptive `title` and `role="button"` attributes

**Review by Hornebom — CHANGES_REQUESTED** [review]

**Comment by Hornebom** [comment]
I feel like on desktop wide viewports the size of a Contributor is too large. 200px should be more than fine. Also the caption text is huge

File: src/components/ContributorsSection.tsx (hunk 1 above)
Comment (onnovisser) [reply src/components/ContributorsSection.tsx:61]: I need the Box to set the size and the Image component doesn't add anything for svgs

File: src/components/ContributorsSection.tsx [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+}
+
+function Contributor({ item }: { item: Item }) {
+  return (
+    <Shelf flexDirection={['row', 'column']} width={['auto', 200, 330]} gap={[4, 2]} role="listitem">
+      <Box width={[150, 'auto']} maxWidth={[150, 'initial']} position="relative" pl={[20, 0]}>
+        <Box borderRadius="50%" overflow="hidden">
+          <Image data={item.image} />
+        </Box>
+        <Box
```
[the remark this answers is not in the document]
Comment (onnovisser) [reply src/components/ContributorsSection.tsx:49]: don't think that's an appropriate use of the `address` element

File: src/components/ContributorsSection.tsx [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+        <Text variant="body1" color="textDisabled">
+          {item.role}
+        </Text>
+      </Stack>
+    </Shelf>
+  )
+}
+
+const MarqueeWrapper = styled(Box)``
+const ButtonWrapper = styled(Box)``
```
[the remark this answers is not in the document]
Comment (onnovisser) [reply src/components/ContributorsSection.tsx:77]: I need their classNames in the `style` element of the `noscript`

File: src/components/ContributorsSection.tsx (hunk 3 above)
Comment (onnovisser) [reply src/components/ContributorsSection.tsx:144]: it renders a `button` element so role is unnecessary, will add title

File: src/components/ContributorsSection.tsx (hunk 2 above)
Comment (onnovisser) [reply src/components/ContributorsSection.tsx:96]: a bit overkill I think

**Comment by Hornebom** [comment]
Some low hanging fruit when running google Lighthouse (Seo and accessibility). Mostly `alt` attributes and color ratio

**Pull request merged**